packages/mollie-integration/src/mollie-payment-strategy.ts

```diff
@@ -69,23 +72,23 @@ export default class MolliePaymentStrategy implements PaymentStrategy {
 
         each(controllers, (controller) => controller.remove());
 
-        const state = this._store.getState();
-        const storeConfig = state.config.getStoreConfig();
+        const storeConfig = this.paymentIntegrationService.getState().getStoreConfig();
```

Comment:
Also you can make it this way:

```
const state = this.paymentIntegrationService.getState();
const storeConfig = state.getStoreConfigOrThrow();
```

And remove condition on line 77, because getStoreConfigOrThrow making the same under the hood:
https://github.com/bigcommerce/checkout-sdk-js/pull/2126/files#diff-e724aa47b68a380801634523464395c8947c6bc130f9268e4f68bfc09702825bR77

Reply:
changed